Mathlib/Probability/Moments.lean

```diff
@@ -184,6 +184,9 @@ theorem mgf_neg : mgf (-X) μ t = mgf X μ (-t) := by simp_rw [mgf, Pi.neg_apply
 
 theorem cgf_neg : cgf (-X) μ t = cgf X μ (-t) := by simp_rw [cgf, mgf_neg]
 
+theorem mgf_smul_left (α : ℝ) : mgf (α • X) μ t = mgf X μ (t * α) := by
+    simp_rw [mgf, Pi.smul_apply, smul_eq_mul, mul_assoc]
```

Comment:
```suggestion
  simp_rw [mgf, Pi.smul_apply, smul_eq_mul, mul_assoc]
```

Reply:
Is there a good reason for reordering the variables? We have alpha to the left of t on one side, and t to the left of alpha on the other side.

Reply:
The reordering is an illusion. After an application of `mgf`, the goal is `∫ (x : Ω), rexp (t * (α • X) x) ∂μ = ∫ (x : Ω), rexp (t * α * X x) ∂μ`. Having said that, I have now switched the statement to `α * t` with an extra application of `mul_comm`